prime-router/settings/organizations.yml

```diff
@@ -473,28 +473,6 @@
         schemaName: "hhsprotect/hhsprotect-covid-19"
         format: "CSV"
       deidentify: true
-    - name: OTC_PROCTORED_NYY
```

Comment:
removed these test receivers since they are now impossible to hit since:
- `test_authorized_for_otc` and `test_authorized_for_home` will always be the same value
- `UNK` is not an allowed value anymore for these fields